lib/queue.js

```diff
@@ -783,6 +783,7 @@ Queue.prototype.processJob = function(job){
 
     return job.moveToCompleted(result).then(function(){
       _this.emit('completed', job, result, 'active');
+      return null;
```

Comment:
what about returning the result from .emit? should be null as well, looks a but nicer :D